client/src/pages/EditEntry.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import UniversalButton from '../components/UniversalButton';
+import { useParams, useHistory } from 'react-router-dom';
+import PageWrapperCenterSpEvenly from '../components/PageWrapperCenterSpEvenly';
+
+const Textarea = styled.textarea`
+  width: 100%;
+  border: 1px solid white;
+  border-radius: 8px;
+  color: ${props => props.theme.colors.background};
+  background-color: ${props => props.theme.colors.quinary};
+  font-family: 'Roboto', sans-serif;
+`;
+
+const Card = styled.form`
+  width: 85%;
+  height: 500px;
+  border-radius: 8px;
+  padding: 10px;
+  font-family: 'Roboto', sans-serif;
+  color: ${props => props.theme.colors.primary};
+  background-color: ${props => props.theme.colors.quinary};
+  opacity: 0.75;
+  margin: 15px;
+  overflow: scroll;
+`;
+const ButtonWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: 4px;
+  justify-content: flex-start;
+  align-items: space-around;
+  margin-top: 8px;
+`;
+
+function EditEntry() {
+  const [edit, setEdit] = React.useState([]);
+  const { editId } = useParams();
+  const history = useHistory();
+
+  async function saveEditedEntriesToDB(value) {
+    setEdit();
+    await fetch(`/api/entries${editId}`, {
```

Comment:
```suggestion
    await fetch(`/api/entries/${editId}`, {
```

I think  the URL is invalid without the slash

Reply:
Thank you.